File: Sakarya.Web.Api/Entities/Concrete/ClientModel.cs
using System;
using Core.Entities;

namespace Entities.Concrete
{
    public class ClientModel : DocumentDbEntity
    {
        public string name { get; set; }
        public int likeCount { get; set; }
        public int quoteCount { get; set; }
        public int replyCount { get; set; }
        public int retweetCount { get; set; }
        public string text { get; set; }
        public string original_text { get; set; }
        public int user_sentiment { get; set; }
        public int createdAt { get; set; }
    }
}

File: Dashboard.Angular/src/app/models/client-model.ts

export class ClientModel {
    name: string;
    likeCount: number;
    quoteCount: number;
    replyCount: number;
    retweetCount: number;
    original_text: string;
    user_sentiment: string;
    createdAt: Number;
}

export class ClientsRequestModel {
    message: string;
    success: boolean;
    data: ClientModel[]
}

export class ClientsRequestCountModel {
    message: string;
    success: boolean;
    data: number
}
